packages/nvidia_nat_harbor/src/nat_harbor/agents/installed/nemo_agent_run_wrapper.py

```
@@ -0,0 +1,227 @@
+
+    async with WorkflowBuilder.from_config(config) as builder:
+        session_manager = await SessionManager.create(config=config, shared_builder=builder)
+        async with session_manager.session(user_id="harbor") as session:
+            async with session.run(instruction) as runner:
+                intermediate_task = None
+                if trajectory_path:
+                    try:
+                        from nat.builder.runtime_event_subscriber import pull_intermediate
+
+                        intermediate_task = asyncio.ensure_future(pull_intermediate())
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)